docs/src/main/sphinx/release/release-476.md

```diff
@@ -0,0 +1,195 @@
+* Fix rare bug when server can hang under load. ({issue}`25816`)
+* Fix regression that prevented graceful shutdown from working. ({issue}`25690`)
+* Fix potential query failure when `fault_tolerant_execution_runtime_adaptive_partitioning_enabled` 
+  is set to `true`. ({issue}`25870`)
+* Fix failure for queries involving casts with `row` types. ({issue}`25864`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Docker image
+
+## CLI
+
+## BigQuery connector
+
+* {{breaking}} Require the `--sun-misc-unsafe-memory-access=allow` JVM configuration 
+  option to run Trino with the connector. ({issue}`25669`)
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Delta Lake connector
+
+* Add support for the `FOR TIMESTAMP AS OF` clause. ({issue}`21024`)
+* Allow configuring *User-ManagedIdentity* in AzureFS. ({issue}`23447`)
+* Add signer type support to the native S3 filesystem. ({issue}`25820`)
+* Avoid skipping statistics computation for all columns when some column types 
+  don't support statistics. ({issue}`24487`)
+* Improve compatibility with S3-compliant storage systems. ({issue}`25791`)
+* Improve query planning performance. ({issue}`24570`)
+* Improve performance when reading tables. ({issue}`25826`)
+* Reduce S3 throttling failures. ({issue}`25781`)
+* Fix failure when reading `variant` type column after executing `optimize` 
+  procedure. ({issue}`25666`)
+* Fix query failures when attempting to read `date` columns stored as integer
+  values in Parquet files. ({issue}`25667`)
+* Fix `SHOW SCHEMAS` failure when catalog names contain mixed case letters. ({issue}`25701`)
+
+## Druid connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## DuckDB connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Elasticsearch connector
+
+## Exasol connector
+
+* Improve performance of selective joins for federated queries. ({issue}`25123`)
+
+## Faker connector
+
+## Google Sheets connector
+
+* Add support for authentication using delegated user credentials with the 
+  `gsheets.delegated-user-email` configuration property. ({issue}`25746`)
+
+## Hive connector
+
+* Add support for excluding certain tables from the directory listing cache 
+  with the `hive.file-status-cache.excluded-tables` configuration property. ({issue}`25715`)
+* Add signer type support to the native S3 filesystem. ({issue}`25820`)
+* Improve compatibility with S3-compliant storage systems. ({issue}`25791`)
+* Allow configuring User-ManagedIdentity in AzureFS. ({issue}`23447`)
+* Improve the directory listing cache invalidation behavior to avoid invalidating the entire
+  cache when dropping a partitioned table. ({issue}`25749`)
+* Fix query failure when reading ORC files with a large row count. ({issue}`25634`)
```

Comment:
FIx a bug where queries could fail with `HIVE_CANNOT_OPEN_SPLIT` error code when reading tables using ORC file format

Reply:
Thanks. I adjusted it slightly:

> * Fix query failures with `HIVE_CANNOT_OPEN_SPLIT` error when reading ORC files with a large row count.